suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -181,11 +181,9 @@ const discoverNetworkBatchThunk = createThunk(
 
         const chunkBundle: Array<DiscoveryItem> = [];
 
-        A.makeWithIndex(batchSize, index => {
-            const accountPath = network.bip43Path.replace(
-                'i',
-                (lastDiscoveredAccountIndex + index).toString(),
-            );
+        A.makeWithIndex(batchSize, batchIndex => {
+            const index = lastDiscoveredAccountIndex + batchIndex;
```

Comment:
this index is used also on line L#198 - the fix is that chunkBundle has to use global index, not batchIndex